OpenRA.Mods.Common/Traits/DamagedByTerrain.cs

```diff
@@ -1,4 +1,4 @@
-﻿#region Copyright & License Information
+#region Copyright & License Information
```

Comment:
This removes a BOM - automated change by VS although from what I see, that's how it should be anyway.